services/comparison.py

```diff
@@ -718,7 +718,6 @@ def head_report(self):
 
     @cached_property
     def has_different_number_of_head_and_base_sessions(self):
-        self.validate()
```

Comment:
already calling validate in the try/catch, so this is just adding 1 extra API call we don't need